src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -509,6 +511,7 @@ export class EditDataGridPanel extends GridParentComponent {
 					return this.removeRow(currentNewRowIndex);
 				}).then(() => {
 					this.newRowVisible = false;
+					this.currentEditCellValue = undefined;
```

Comment:
It seems like this should be done inside resetCurrentCell() instead. Can you see any reason why it wouldn't be done there? There's a couple other places we're calling that method and it seems like in all those cases we should also be clearing the currentEditCellValue (such as https://github.com/microsoft/azuredatastudio/blob/master/src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts#L512)

Maybe not something to take for the 1.15 release but worth checking while we're fixing this.

Reply:
I agree, at least two other times its done like that.